rfcs/QueryLevelNullability.md

```diff
@@ -170,5 +170,19 @@ This syntax consciously does not cover the following use cases:
   that says "if a value would be `null` make it some other value instead". We are
   not interested in covering that in this proposal.
   
-## Implementation
-GraphQL.js: https://github.com/graphql/graphql-js/pull/2824
+## Work Items
+Patches that will need to be made if this proposal is accepted. The 
+[RFC proposal process](https://github.com/graphql/graphql-spec/blob/main/CONTRIBUTING.md)
+requires that a proof of concept is implemented in a GraphQL library. As work items are completed,
+PRs will be linked here.
+- Spec Changes
+- Official Libraries
+  - GraphQL.js: https://github.com/graphql/graphql-js/pull/2824
```

Comment:
let's double check with @xuewei8910 to make sure this PR is ready for external eyes (I think he mentioned it needed to be updated with latest)